ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml

```diff
@@ -24,16 +24,27 @@ releases:
       version: "4.15"
 resources:
   '*':
-    limits:
-      memory: 4Gi
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: unit
-  commands: echo "test"
-  container:
-    from: src
+- always_run: false
+  as: e2e-cluster-density-v2-single-node
```

Comment:
Just to reduce the number of jobs we have to create, could we just have this be `e2e-kube-burner-single-node`?  and we iterate through the tests defined (cluster-density-v2, node-density-*, etc) ?

Reply:
Are you suggesting a chain with all kube-burner tests? If yes, I have intentionally kept this separate, so that we don't end up triggering all the other tests in kube-burner when we have made changes only in cd-v2. I am open to other thoughts too.